src/main/java/dev/jbang/Util.java

```diff
@@ -844,4 +844,12 @@ public static boolean isURL(String str) {
 		return Optional.empty();
 	}
 
+	public static void deleteDirectory(File directory) {
```

Comment:
This is not necessary, there's already a method for this: `Util.deleteFolder()`

Reply:
Thank you! I changed.